src/main/java/duke/Task.java

```diff
@@ -0,0 +1,75 @@
+package duke;
+
+/**
+ * Stores a Task object.
+ */
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    /**
+     * Instantiates a Task.
+     *
+     * @param description Name of the task.
+     */
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    /**
+     * Instatiates a task with completion status.
+     *
+     * @param description Name of the task.
+     * @param isDone      Completion status of the task.
+     */
+    public Task(String description, boolean isDone) {
+        this.description = description;
+        this.isDone = isDone;
+    }
+
+    /**
+     * Returns an visual representation of the completion
+     * status of the task.
+     *
+     * @return Tick or cross symbol.
+     */
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718");
```

Comment:
Maybe can consider adding a comment here to describe what the unicode characters represent